backend/config/config.go

```diff
@@ -59,9 +59,15 @@ type proxyConfig struct {
 	ProxyAddress string `json:"proxyAddress"`
 }
 
+type blockExplorerConfig struct {
+	Enabled bool   `json:"useCustomBlockExplorer"`
+	URL     string `json:"blockExplorerURL"`
+}
+
 // Backend holds the backend specific configuration.
 type Backend struct {
-	Proxy proxyConfig `json:"proxy"`
+	Proxy            proxyConfig         `json:"proxy"`
+	BTCBlockExplorer blockExplorerConfig `json:"blockExplorer"`
```

Comment:
please move this to `btcCoinConfig`, which allows extending this feature to testnet and alts in the future.